Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.2.5
+FROM ruby:2.5.0
```

Comment:
```suggestion
FROM ruby:2.5
```
Allow the patch version to increase to get any ruby improvements / fixes

Reply:
Thank you @camallen ! really appreciated, I've applied that change now 👍